src/server.tsx

```diff
@@ -243,22 +181,25 @@ const buildBannerData = async (
     }
 };
 
-// Pre-ES module endpoints (expected to be removed once module approach is validated)
-app.get(
+// TODO replace with module-friendly solution
+/* app.get(
```

Comment:
removable now?

Reply:
I was keen to keep this as a reminder that we should replace it. (Albeit that is a larger task so wanted to do in a separate PR.)